### PR TITLE
FH-3084 Expose MongoDB client

### PR DESCRIPTION
**Pull request by aidenkeating**
This exposes the MongoDB Db object created during during
initialization.

### CODE REVIEW

File: lib/fhmongodb.js [hunk 1]
```diff
@@ -30,6 +30,17 @@ var Database = function (host, port, pDriverOptions, retryWaitTime, dbname) {
 
 util.inherits(Database, EventEmitter);
 
+/**
+ * Get the MongoDB database instance.
+ *
```
Comment (fh-qube) [inline lib/fhmongodb.js]: MINOR Remove the useless trailing whitespaces at the end of this line. [rule](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3ATrailingWhitespace)

File: lib/fhmongodb.js [hunk 2]
```diff
@@ -30,6 +30,17 @@ var Database = function (host, port, pDriverOptions, retryWaitTime, dbname) {
 
 util.inherits(Database, EventEmitter);
 
+/**
+ * Get the MongoDB database instance.
+ * 
+ * Will return `null` if connection has not been established.
+ *
```
Comment (fh-qube) [inline lib/fhmongodb.js]: MINOR Remove the useless trailing whitespaces at the end of this line. [rule](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3ATrailingWhitespace)

**Comment by fh-qube** [comment]
SonarQube analysis reported 2 issues:
* MINOR 2 minor

Watch the comments in this conversation to review them.

**Comment by aidenkeating** [comment]
@david-martin @danbev Could you take a look please?

File: test/test_fhmongodb.js [hunk 3]
```diff
@@ -418,3 +418,25 @@ exports[ 'test update and delete item with an id that is not hex' ] = function (
...
+  self.db.name = "test-fhmongodb-database";
+
+  self.db.on('tearUp', do_ops);
+  // Should be null before tearUp.
+  assert(!self.db.getMongoClient());
+
+  function do_ops() {
+    self.db.db.authenticate(config.database.adminauth.user, config.database.adminauth.user, {authSource:"admin"}, function(err, result) {
+      // Should not be null after tearUp.
+      assert(self.db.getMongoClient());
```
Comment (david-martin) [inline test/test_fhmongodb.js]: could add `assert.equal(self.db.db, self.db.getMongoClient())`

**Comment by aidenkeating** [comment]
@david-martin I have updated based on your suggestions.

**Pull request merged**